app/konnect-platform/teams-and-roles.md

```diff
@@ -89,7 +89,7 @@ rows:
   - team: Analytics Viewer
     description: Users can view the [Analytics](/advanced-analytics/) summary and report data.
   - team: Organization Admin
-    description: Users can fully manage all entities and configuration in the organization.
+    description: Users can fully manage all entities and configuration in the organization. Each organization has one Owner, that always has this role and is the only user who can delete the organization.
```

Comment:
```suggestion
    description: Users can fully manage all entities and configuration in the organization. In addition to users granted the Organization Admin role, each organization also has one Owner, who always has this role and is the only user who can delete the organization.
```
Trying to make it clear that the owner is just one user type, and isn't the only user that can have an admin role.